app/repositories/events/LocationsWithVenuesRepository.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories.events
+
+import java.io.InputStream
+import java.util
+
+import com.github.ghik.silencer.silent
+import config.MicroserviceAppConfig
+import model.persisted.eventschedules.{Location, Venue}
+import org.yaml.snakeyaml.Yaml
+import play.api.Play
+import play.api.libs.json.{Json, OFormat}
+import resource._
+
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.language.postfixOps
+
+
+case class LocationWithVenue(name: String, venues: List[Venue])
```

Comment:
Should be put this in model.persisted? Or do we want to have the persisted model with the corresponding repo in same file?